src/contest.c

```diff
@@ -4538,29 +4655,29 @@ static void task08_080CD1CC(u8 taskId)
     if (sprite->pos2.x == -70)
     {
         sprite->invisible = TRUE;
-        sContest.unk1920A_6 = 0;
+        eContest.applauseMeterIsMoving = FALSE;
         DestroyTask(taskId);
     }
 }
 
-static void sub_80DDCDC(s8 a)
+static void ShowAndUpdateApplauseMeter(s8 unused)
 {
-    u8 taskId = CreateTask(sub_80DDD20, 5);
+    u8 taskId = CreateTask(Task_ShowAndUpdateApplauseMeter, 5);
 
-    gTasks[taskId].data[0] = a;
-    sContest.unk1920A_5 = 1;
+    gTasks[taskId].data[0] = unused;
```

Comment:
`data[0]` may not be used but the local variable here is.

Reply:
But someone looking at the arguments to the function will want to know that the argument doesn't matter at all, right?